designs/0032-stan-output-formats.md

```diff
@@ -0,0 +1,294 @@
+- Feature Name: stan-output-formats
+- Start Date: 2022-01-11
+- RFC PR:
+- Stan Issue:
+
+# Summary
+[summary]: #summary
+
+In order to monitor the progress of CmdStan's inference algorithms,
+the outputs from CmdStan must be available to downstream readers on a streaming basis.
+This design provides an alternative to the [Stan CSV file format](https://mc-stan.org/docs/cmdstan-guide/stan_csv.html)
+for outputs of the Stan inference algorithms which are exposed by the `stan::services` layer.
```

Comment:
Can you add one more sentence here describing more specifically what your alternative is? It would just help me understand where the motivation section is going towards

Reply:
added more to the summary as well.